draft-ietf-radext-radiusv11.md

```diff
@@ -336,7 +336,7 @@ Values
 >>
 >> Server Behavior
 >>
->>> If the server receives no ALPN name from the client, it MAY reply with a TLS alert of "no_application_protocol" (120).  It MUST then close the TLS connection.
+>>> If the server receives no ALPN name from the client, it MUST close the TLS connection. It MAY reply with a TLS alert of "no_application_protocol" (120) prior to closing, see below for further discussion on this.
```

Comment:
```suggestion
>>> If the server receives no ALPN name from the client, it MUST close the TLS connection, but it MAY reply with a TLS alert of "no_application_protocol" (120) prior to closing, see below for further discussion on this.
```

This way it is in the same sentence, so hopefully people will not stop reading after "MUST close the connection"

Reply:
sounds good, thanks.